ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/worldstate/WorldStatePeerTrieNodeFinder.java

```diff
@@ -90,7 +98,10 @@ public Optional<Bytes> getAccountStorageTrieNode(
       return cachedValue;
     }
     final Optional<Bytes> response =
-        findByGetNodeData(Hash.wrap(nodeHash))
+        // Call findByGetNodeData only if protocol version < eth 67
+        (protocolManager.getHighestProtocolVersion() < EthProtocolVersion.V67
+                ? findByGetNodeData(Hash.wrap(nodeHash))
```

Comment:
Does the Fastsync RequestDataStep also need such a check? It also uses GetNodeData https://github.com/hyperledger/besu/blob/main/ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/RequestDataStep.java#L48